### PR TITLE
Buffs the random virus event

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

Random virus event was just sort of sad. You'd get the flu and nothing would happen. Worst case scenario, brainrot.

This changes it up so that the viruses can be very very bad. Beesease, Ice 9, Retrovirus, DNA randomizer
#### Changelog

:cl:
rscadd: The virus random event now has some teeth as compared to before. Be on the lookout for more virulent outbreaks, medbay staff.
/:cl:

### CODE REVIEW

**Comment by Cruix** [comment]
Please make better commit messages, it is entirely possible that people will be reading those years from now and need to know what that commit did.

File: code/modules/events/disease_outbreak.dm [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 /datum/round_event/disease_outbreak/start()
 	if(!virus_type)
-		virus_type = pick(/datum/disease/dnaspread, /datum/disease/advance/flu, /datum/disease/advance/cold, /datum/disease/brainrot, /datum/disease/magnitis)
+		virus_type = pick(/datum/disease/dnaspread, /datum/disease/fluspanish, /datum/disease/cold9, /datum/disease/brainrot, /datum/disease/magnitis, /datum/disease/dna_retrovirus, /datum/disease/beesease, )
```
Comment (Super3222) [inline code/modules/events/disease_outbreak.dm]: the end of that list doesn't need a ,

**Comment by ShadowDeath6** [comment]
> no xeno regression virus
> no monkey virus

fuck off adam

**Comment by Cruix** [comment]
<sub><sub><sub><sub><sub><sub><sub>Bring back monkey mode</sub></sub></sub></sub></sub></sub></sub>

**Comment by ghost** [comment]
i might make another event that randomly releases the worst viruses but that'd have to trigger far less oftne than this

**Comment by ghost** [comment]
No. This will be too disruptive to the round. Micro-events should not have the power to end the round.
 Maybe if you could explain where the fun comes from when everyone is dying of diseases then I might change my mind.

**Comment by ghost** [comment]
yeah no, i'm not even going to bother explaining to you if you don't understand the whole thing of 'random events that are actually dangerous yet beatable' instead of ranging from 'completely harmless' like scrubber fuckups and radstorms to 'fuck the round' like meteors

this is fine and your opinion is valued oh wait no it's not

**Pull request merged**